assets/src/dashboard/animations/stories/blinkOn.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { AnimatorOutput, AnimationOutput, WithAnimation } from '../animator';
+import { ANIMATION_TYPE } from '../constants';
+import getAnimationConfigs from '../configs';
+import getInitialStyleFromKeyframes from '../utils/getInitialStyleFromKeyframes';
+
+export default {
+  title: 'Dashboard/Animations/Blink On',
```

Comment:
nit: title should be the same as the component, so no space, just `BlinkOn`

Reply:
Okay, I'll remove the same :D